src/Apps/Artwork/Components/__tests__/ArtworkBanner.test.tsx

```diff
@@ -0,0 +1,90 @@
+import {
+  ArtworkAuctionBannerFixture,
+  ArtworkFairBannerFixture,
+  ArtworkUpcomingShowBannerFixture,
+  ArtwrorkNoBannerFixture,
+} from "Apps/__test__/Fixtures/Artwork/ArtworkBanner"
+import { ArtworkBannerFragmentContainer } from "Apps/Artwork/Components/ArtworkBanner"
+import { renderRelayTree } from "DevTools"
+import { GraphQLResolveInfo } from "graphql"
+// import { cloneDeep } from "lodash"
+import { graphql } from "react-relay"
+
+jest.unmock("react-relay")
+
+describe("ArtworkBanner", () => {
+  const getWrapper = async (response = ArtwrorkNoBannerFixture) => {
+    return await renderRelayTree({
+      Component: ArtworkBannerFragmentContainer,
+      query: graphql`
+        query ArtworkBanner_Test_Query {
+          artwork(id: "richard-anuszkiewicz-lino-yellow-318") {
+            ...ArtworkBanner_artwork
+          }
+        }
+      `,
+      mockResolvers: {
+        Artwork: () => ({
+          ...response,
+          // FIXME: Need to figure out how to return proper values for aliesed objects
+          context: (_source, _args, _context, info: GraphQLResolveInfo) => {
+            const alias = info.fieldNodes[0].alias!.value
+            const x = response[alias]
+            return x
+          },
```

Comment:
For context, I’m surprised that graphql-tools’ mocking doesn’t look for aliases when returning a static payload, so need to figure out if that’s supposed to work or not